apk-generator/v2/requirements.txt

```diff
@@ -8,4 +8,5 @@ android-asset-resizer
 pillow>=3.4.2,<3.5
 Flask-HTMLmin>=1.2,<2
 validators>=0.11.2,<0.12
-bleach>=1.5.0,<2
\ No newline at end of file
+bleach>=1.5.0,<2
+flask-socketio>=2.9.0
```

Comment:
`eventlet` is also needed. Since we'll be using that as the worker with gunicorn.